src/gigs-board/components/layout/SearchByAuthor.jsx

```diff
@@ -59,15 +59,54 @@ const onChangeAuthor = (selectedAuthors) => {
   onSearchAuthor(selectedAuthors[0]?.name);
 };
 
+function customMenu(results, menuProps, menuState) {
+  // return (
+
+  // );
+
+  return (
+    <div
+      // style={{ width: "10rem", maxHeight: "10rem", overflowY: "scroll" }}
+      // id="basic-typeahead-single"
+      aria-label="menu-options"
+      class="rbt-menu dropdown-menu show"
+      role="listbox"
+      // style="position: absolute; inset: 0px auto auto 0px; display: block; max-height: 300px; overflow: auto; transform: translate3d(0px, 38px, 0px); width: 160px;"
```

Comment:
I feel that this is still a draft PR given that there are some commented-out parts in the code.